src/zenml/integrations/slack/alerters/slack_alerter.py

```diff
@@ -144,7 +147,7 @@ def _get_disapprove_msg_options(
 
     def _create_blocks(
         self, message: str, params: Optional[BaseAlerterStepParameters]
-    ) -> List[Dict]:  # type: ignore
+    ) -> List[Dict]: #type: ignore
```

Comment:
is this change needed?

Reply:
I have a feeling that this is what's causing the linting to fail at the moment.

Reply:
Sorry I didn't mean to change that. Have rerun the `format.sh` file and it's been reverted. My bad!